app/controllers/api/v1/statuses/reblogs_controller.rb

```diff
@@ -32,4 +32,8 @@ def status_for_reblog
   def status_for_destroy
     current_user.account.statuses.where(reblog_of_id: params[:status_id]).first!
   end
+
+  def reblog_params
+    params.permit(:visibility)
```

Comment:
Agree direct and limited make no sense. Limited might make sense if explicit group addressing is added to the API, but right now, it should be excluded. Amended. Public/unlisted is fine. Us excluding stuff from the PTLs is a readability decision rather than a spec thing. A different platform might want to show all public boosts in a stream.